src/components/swap/SwapButtons/SwapButtons.tsx

```diff
@@ -46,6 +46,18 @@ interface SwapButtonsProps {
   setWrapState: ((wrapState: WrapState) => void) | undefined
 }
 
+const errorBox = ({
```

Comment:
This is a component. Makes sense to use the title case. Besides that, I will make `swapErrorMessage` a required prop :)

Reply:
yup, true, it is a component. What do you mean by the title case?

Reply:
`ErrorBox` :)

Reply:
ok, you mean to rename it as ErrorBox! :D

Reply:
Renamed, moved to a separate file, and leave prop `swapErrorMessage` as a string or undefined, and both cases as undefined or empty string are handled. If it will be always a required prop so we can easily change it.